rllib/algorithms/ppo/tf/ppo_tf_rl_module.py

```diff
@@ -84,8 +86,8 @@ def _forward_train(self, batch: NestedDict):
 
         return output
 
-    @override(PPORLModule)
-    def _compute_values(self, batch, device=None):
+    @override(ValueFunctionAPI)
+    def compute_values(self, batch: Dict[str, Any]) -> TensorType:
         infos = batch.pop(Columns.INFOS, None)
         batch = tree.map_structure(lambda s: tf.convert_to_tensor(s), batch)
```

Comment:
Tf doesn't really need to do any `.to([some device]` when processing tensors. Unlike torch, it can handle CPU inputs if the model is on the GPU (torch can't and requires you to explicitly move the tensor to GPU first). So I think, this simplification here is ok. We will soon get rid of tf support altogether.